examples/window.rs

```diff
@@ -670,9 +695,58 @@ impl WindowState {
         self.window.set_cursor(cursor);
     }
 
+    /// Custom cursor from an URL.
+    #[cfg(web_platform)]
+    fn url_custom_cursor(&mut self, event_loop: &ActiveEventLoop) {
+        use winit::platform::web::CustomCursorExtWebSys;
+
+        let cursor = CustomCursor::from_url(
+            format!(
+                "https://picsum.photos/128?random={}",
+                URL_COUNTER.fetch_add(1, Ordering::Relaxed)
+            ),
+            64,
+            64,
+        );
+        let cursor = event_loop.create_custom_cursor(cursor);
+
+        self.window.set_cursor(cursor);
+    }
+
+    /// Custom cursor from a URL.
+    #[cfg(web_platform)]
+    fn animation_custom_cursor(
+        &mut self,
+        event_loop: &ActiveEventLoop,
+        custom_cursors: &[CustomCursor],
+    ) {
+        use std::time::Duration;
+        use winit::platform::web::CustomCursorExtWebSys;
+
+        let cursor = CustomCursor::from_animation(
+            Duration::from_secs(3),
+            vec![
+                custom_cursors[0].clone(),
+                custom_cursors[1].clone(),
+                event_loop.create_custom_cursor(CustomCursor::from_url(
+                    format!(
+                        "https://picsum.photos/128?random={}",
```

Comment:
Could we use url to github for that? Like we have cursors laying around in the repo already in png formats, so they should be fine.

Reply:
Well the point is to always have it be a different picture, so you can see what happens when its not loading immediately.